dashboard/app/models/user.rb

```diff
@@ -891,20 +891,14 @@ def recent_courses_and_scripts
     user_scripts = in_progress_and_completed_scripts.
       select {|user_script| !course_scripts_script_ids.include?(user_script.script_id)}
 
-    # TODO: replace courseName with name (here and in components) as some of these
-    # are courses and some are scripts
-    # TODO: may make more sense to generate links on the client (might need to
-    # provide both name and title to accomplish this). If we do we can get rid of
-    # include Rails.application.routes.url_helpers
-    # TODO: assigned sections seems like it belongs somewhere else
-    # TODO: currently no way for levelbuilders to edit description_short
-
     course_data = courses.map do |course|
       {
-        courseName: data_t_suffix('course.name', course[:name], 'title'),
+        name: data_t_suffix('course.name', course[:name], 'title'),
         description: data_t_suffix('course.name', course[:name], 'description_short'),
         link: course_path(course),
         image: '',
+        # assigned_sections is current unused. When we support this, I think it makes
+        # more sense to get/store this data separately from courses.
```

Comment:
I don't think that's entirely unreasonable, and might ultimately be the best approach. However, it doesn't feel quite right to me - though I'm struggling a little to justify why not.

I think part of it is that right now all of our course/script data is not user dependant. We use the current user to decide which set of courses/scripts to return data for, but the actual data is user agnostic. Adding assigned_courses here would change that.

I will continue to noodle on this.